pkgs/development/python-modules/hledger-utils/default.nix

```
@@ -0,0 +1,66 @@
+{lib
+, buildPythonPackage
+, fetchFromGitLab
+, setuptools
+, setuptools-scm
+, unittestCheckHook
+, hledger
+, perl
+, rich
+, pandas
+, scipy
+, psutil
+, matplotlib
+, drawilleplot
+, asteval
+}:
+
+buildPythonPackage rec {
+  pname = "hledger-utils";
+  version = "1.12.1";
+
+  format = "pyproject";
+
+  src = fetchFromGitLab {
+    owner = "nobodyinperson";
+    repo = "hledger-utils";
+    rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```

To prevent tag/branch confusion.